packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -604,6 +605,10 @@ export default {
       description:
         'Explicitly set device to use by name.  The value is not required if you have a single device connected.',
     },
+    {
+      name: '--destination <string>',
+      description: 'Explicitly extend distination e.g. "arch=x86_64"',
```

Comment:
**suggestion**: do you mean `destination` instead of `distination`

Reply:
https://github.com/react-native-community/cli/pull/2025

Reply:
Sorry, been away and missed this. Thanks for the PR 👍